### ISSUE TITLE
Script failure on execution MIME

### ISSUE THREAD

**Issue by ryanmusante** [magiruuvelvet/LightweightQtDRMStreamViewer#1]
Compiled and run with error stating the following ::

```
 Netflix Player detected! Injecting Netflix 1080p Unlocker...js: Refused to execute script from 'https://assets.nflxext.com/en_us/ffe/player/html/cadmium-playercore-5.0008.803.011.js' because its MIME type ('text/plain') is not executable, and strict MIME type checking is enabled.
```

**Comment by omni6** [comment]
Same here. App works well with `urlInterceptor:false`  but then not in HD. With enabled interceptor i get Netflix error [UI3010](https://help.netflix.com/en/node/61478)

**Comment by magiruuvelvet** [comment]
I managed to fix it and it works now. Pushing a commit soon.

Here is a screenshot for proof.
![screenshot_20180328_093528](https://user-images.githubusercontent.com/13280758/38015250-7eb0a9ea-326b-11e8-8ddf-75ac4ffb7404.png)

The problem was the MIME type. Chromium (the one which Qt uses under the hood) refuses to execute plain text files. `raw.githubusercontent.com` apparently returns the wrong MIME type. I'm using Rawgit for now. I may mirror the script on my own servers though if it breaks again.

**Comment by magiruuvelvet** [comment]
Fixed in e4ea4599a694f8d3480216b5de315a7dec0b205b
